compass/ocean/tests/internal_wave/ten_day_test/namelist.forward

```diff
@@ -0,0 +1,10 @@
+config_dt = '000:05:00'
+config_btr_dt = '000:00:15'
+config_time_integrator = 'split_explicit'
+config_run_duration = '0010_00:00:00'
+config_use_cvmix = .true.
+config_pio_stride = 4
+config_use_mom_del2 = .true.
+config_mom_del2 = 10.0
+config_implicit_bottom_drag_coeff = 1.0e-2
+config_use_cvmix_convection = .true.
```

Comment:
```suggestion
config_implicit_bottom_drag_coeff = 1.0e-2
config_use_cvmix_convection = .true.
config_AM_globalStats_enable = .true.
config_AM_globalStats_compute_on_startup = .true.
config_AM_globalStats_write_on_startup = .true.
config_AM_globalStats_text_file = .true.
```
You may have good reason for not including global stats in this run, but the legacy test case includes it and sometimes it's useful.  I'd include it if you don't have a reason not to.